client/verta/tests/test_versioning/test_dataset.py

```diff
@@ -9,9 +9,9 @@ def test_s3_bucket(self):
         pytest.importorskip("boto3")
 
         dataset = verta.dataset.S3("s3://verta-starter")
-        assert len(dataset._msg.s3.components) > 1
```

Comment:
Skip this file when reviewing; it's just a find-and-replace.

Reply:
Unfortunately, it's a bit more complex than a find-and-replace now, but it's fairly trivial and they all still pass.